includes/amp-helper-functions.php

```diff
@@ -1707,6 +1707,7 @@ function amp_get_content_sanitizers( $post = null ) {
 	// Force core essential sanitizers to appear at the end at the end, with non-essential and third-party sanitizers appearing before.
 	$expected_final_sanitizer_order = [
 		AMP_Auto_Lightbox_Disable_Sanitizer::class,
+		AMP_PWA_Script_Sanitizer::class, // Must come before core theme sanitizer as it can contains offline/500 templates.
```

Comment:
Why does it need to come before `AMP_Core_Theme_Sanitizer`? I should think the only key thing is that it appears before `AMP_Script_Sanitizer`, to prevent the `script` tags from getting removed.